docs/1.getting-started/6.data-fetching.md

```diff
@@ -537,3 +537,24 @@ const { data } = await useFetch('/api/superjson', {
 })
 </script>
 ```
+
+### Adding types to the builtin serializer
+
+Since [Nuxt v3.4](https://nuxt.com/blog/v3-4#payload-enhancements), it is possible to define your own serializer/deserializer for types that are not supported by Nuxt.
+
+In the example below, we define a serializer for the [Luxon](https://moment.github.io/luxon/#/) DateTime class. Note that the "DateTime" string in the first argument is the type identifier and must be the same on both ends. 
+```ts [plugins/dateTimeType.server.ts]
+export default definePayloadPlugin(() => {
+  definePayloadReducer("DateTime", (value) => {
```

Comment:
These can/should be in the same plugin, by the way. Nuxt will strip out server/client code when compiling it.

See example in https://nuxt.com/docs/api/composables/use-nuxt-app#payload.